client/src/constants/routes.js

```diff
@@ -11,3 +11,4 @@ export const WORKBENCH = '/workbench';
 export const USER_GUIDE = '/user-guide';
 export const ONBOARDING = '/onboarding';
 export const ACCOUNT = '/account';
+export const SEARCH = '/workbench/search';
```

Comment:
Can you double check with @nathanfranklin? I think site search may need to be added in some form to the setup_complete check we do. Like, CMS and public data should do something, but My Data, etc. may be broken if a user doesn't have those systems in existence yet due to setup not finishing the creation?

Reply:
will the final route be '/workbench/search'  or `/search`? 

Currently, it is `/search` (and this `SEARCH` is unused). 

------------------------------

If it is a guest user, then the API only returns search results for the CMS 👍 .  If the user is logged in, then the API returns results for the CMS + Community Data.  In that case, the `setupComplete` should probably be queried as well.  I'll add a comment there.